tsconfig.json

```diff
@@ -62,7 +62,6 @@
     "src/components/VLogoLoader/**.vue",
     "src/components/VMediaInfo/**.vue",
     "src/components/VMediaTag/**.vue",
```

Comment:
Cannot add `VMetaSearch` to `tsconfig` anymore because it depends on Popover and Reakit.